corehq/apps/sms/util.py

```diff
@@ -169,20 +169,21 @@ def clean_text(text):
     return text
 
 
-def get_contact(contact_id):
-    from casexml.apps.case.models import CommCareCase
+def get_contact(domain, contact_id):
+    from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+    from corehq.form_processor.exceptions import CaseNotFound
     contact = None
     try:
-        contact = CommCareCase.get(contact_id)
-    except ResourceNotFound:
+        contact = CaseAccessors(domain).get_case(contact_id)
+    except (ResourceNotFound, CaseNotFound):
         pass
 
-    if contact and contact.doc_type == 'CommCareCase':
+    if contact and contact.doc_type == 'CommCareCase' and contact.domain == domain:
```

Comment:
the reason i couldn't use is_commcarecase is because for the couch version of the cases, it would always return true even if the underlying doc wasn't a case.  (since we're just calling .get with an id i wanted to make sure it wasn't some other non-case doc that was found)